graylog2-web-interface/webpack/vendor-module-ids.json

```diff
@@ -921,7 +921,103 @@
+      "node_modules/babel-runtime/node_modules/core-js/library/modules/es6.symbol.js": 146,
+      "node_modules/babel-runtime/node_modules/core-js/library/modules/_meta.js": 147,
+      "node_modules/babel-runtime/node_modules/core-js/library/modules/_enum-keys.js": 148,
+      "node_modules/babel-runtime/node_modules/core-js/library/modules/_is-array.js": 251,
+      "node_modules/babel-runtime/node_modules/core-js/library/modules/_object-gopn-ext.js": 252,
+      "node_modules/babel-runtime/node_modules/core-js/library/modules/es6.object.to-string.js": 253,
+      "node_modules/babel-runtime/node_modules/core-js/library/modules/es7.symbol.async-iterator.js": 254,
+      "node_modules/babel-runtime/node_modules/core-js/library/modules/es7.symbol.observable.js": 255,
+      "node_modules/babel-runtime/node_modules/core-js/library/fn/object/set-prototype-of.js": 256,
+      "node_modules/babel-runtime/node_modules/core-js/library/modules/es6.object.set-prototype-of.js": 257,
+      "node_modules/babel-runtime/node_modules/core-js/library/modules/_set-proto.js": 258,
+      "node_modules/babel-runtime/node_modules/core-js/library/fn/object/create.js": 259,
+      "node_modules/babel-runtime/node_modules/core-js/library/modules/es6.object.create.js": 260,
+      "node_modules/babel-runtime/node_modules/core-js/library/fn/object/entries.js": 261,
+      "node_modules/babel-runtime/node_modules/core-js/library/modules/es7.object.entries.js": 262,
+      "node_modules/babel-runtime/node_modules/core-js/library/fn/object/values.js": 263,
+      "node_modules/babel-runtime/node_modules/core-js/library/modules/es7.object.values.js": 264,
+      "node_modules/babel-runtime/node_modules/core-js/library/fn/array/from.js": 265,
+      "node_modules/babel-runtime/node_modules/core-js/library/modules/es6.array.from.js": 266,
+      "node_modules/babel-runtime/node_modules/core-js/library/modules/_iter-call.js": 267,
+      "node_modules/babel-runtime/node_modules/core-js/library/modules/_is-array-iter.js": 268,
+      "node_modules/babel-runtime/node_modules/core-js/library/modules/_create-property.js": 269,
+      "node_modules/babel-runtime/node_modules/core-js/library/modules/core.get-iterator-method.js": 270,
+      "node_modules/babel-runtime/node_modules/core-js/library/modules/_classof.js": 271,
+      "node_modules/babel-runtime/node_modules/core-js/library/modules/_iter-detect.js": 272,
+      "node_modules/react-is/cjs/react-is.production.min.js": 273,
+      "node_modules/color-convert/conversions.js": 74,
+      "node_modules/clone/clone.js": 76,
+      "node_modules/buffer/index.js": 79,
+      "node_modules/base64-js/index.js": 80,
+      "node_modules/buffer/node_modules/ieee754/index.js": 84,
+      "node_modules/isarray/index.js": 89,
+      "node_modules/color-convert/index.js": 274,
+      "node_modules/color-convert/route.js": 275,
+      "node_modules/color-string/color-string.js": 277
```

Comment:
I am not seeing these after a `yarn build` locally. Are you sure you did the run with a clean `master` after running `yarn install`?

Reply:
I also get these changes after running `mvn clean compile` using an up-to-date master manifest.

Reply:
@dennisoelkers it was the changes I was seeing. But with recent packages being added/removed in #6648 I will need to redo this build anyway